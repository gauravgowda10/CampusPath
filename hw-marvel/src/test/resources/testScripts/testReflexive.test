# test BFS on reflexive case

# build the graph
LoadGraph g threeHeroes.tsv

# find the shortest path Spiderman to CaptainAmerica
FindPath g Spiderman CaptainAmerica

# find the shortest path CaptainAmerica to Spiderman
FindPath g CaptainAmerica Spiderman

# find the shortest path CaptainAmerica to CaptainAmerica
FindPath g CaptainAmerica CaptainAmerica