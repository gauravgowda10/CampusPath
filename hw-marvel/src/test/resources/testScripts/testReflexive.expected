# test BFS on reflexive case

# build the graph
loaded graph g

# find the shortest path Spiderman to CaptainAmerica
path from Spiderman to CaptainAmerica:
Spiderman to IronMan via IronMan
IronMan to CaptainAmerica via Avengers

# find the shortest path CaptainAmerica to Spiderman
path from CaptainAmerica to Spiderman:
CaptainAmerica to IronMan via Avengers
IronMan to Spiderman via IronMan

# find the shortest path CaptainAmerica to CaptainAmerica
path from CaptainAmerica to CaptainAmerica: